src/components/Searchbar.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+import iconSearch from '../images/icon_magnifying-glass.svg';
+
+export default function Searchbar({ userInput, handleUserInput }) {
+  return (
+    <SearchbarContainer>
+      <div>
+        <img src={iconSearch} alt="search icon" width="26px" height="26px" />
+        <label htmlFor="catfood-search">Suche</label>
+      </div>
```

Comment:
It might be possible to put the `img` element inside of the `label` element in order to avoid the `div`, have you tried?